returnn/torch/distributed.py

```diff
@@ -27,7 +27,7 @@ def __init__(self, config):
         """
         import torch.distributed as dist
 
-        dist.init_process_group("nccl")
+        dist.init_process_group()
```

Comment:
Your PR description says that you create a Gloo backend, but I thought this code here would create both a Gloo and a NCCL backend? See the documentation. This is also important. We really want to have the NCCL backend, so the PR title is misleading? We are not changing the NCCL backend to the Gloo backend. We want to *add* the Gloo backend as an *additional* backend.

You also should add a comment which explains that, and fix the PR title, and the PR description to explain that.

Reply:
Thanks for suggestion. I have added more description and changed the title. Would it be informative enough now?